modules/installation-aws-config-yaml.adoc

```diff
@@ -312,7 +312,16 @@ endif::gov,secret,china[]
 ifdef::gov,secret,china[]
 <1> Required.
 endif::gov,secret,china[]
-<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode, instead of having the CCO dynamically try to determine the capabilities of the credentials. For details about CCO modes, see the _Cloud Credential Operator_ entry in the _Red Hat Operators reference_ content.
+<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode. By default, the CCO uses the root credentials in the `kube-system` namespace to dynamically try to determine the capabilities of the credentials. For details about CCO modes, see _About the Cloud Credential Operator_ in the _Authentication and authorization_ content.
```

Comment:
Quotations for the section title?
```suggestion
<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode. By default, the CCO uses the root credentials in the `kube-system` namespace to dynamically try to determine the capabilities of the credentials. For details about CCO modes, see the "About the Cloud Credential Operator" section in the _Authentication and authorization_ guide.
```

Thinking about this from IBMSG:
> If you refer to a smaller piece of work within a larger container such as a chapter within a book or an article in a magazine, use italic font for the larger container and double quotation marks for the smaller piece.

Same for other similarly-styled citations throughout PR, if changed here.